src/objc_func_selector.c

```diff
@@ -0,0 +1,65 @@
+
+/* Compiler implementation of the D programming language
+ * Copyright (c) 2015 by Digital Mars
+ * All Rights Reserved
+ * written by Jacob Carlborg
+ * http://www.digitalmars.com
+ * Distributed under the Boost Software License, Version 1.0.
+ * http://www.boost.org/LICENSE_1_0.txt
+ * https://github.com/D-Programming-Language/dmd/blob/master/src/objc_func_selector.c
+ */
+
+#include "aggregate.h"
+#include "attrib.h"
+#include "declaration.h"
+#include "expression.h"
+#include "id.h"
+#include "objc.h"
+#include "scope.h"
+
+void objc_FuncDeclaration_semantic_setSelector(FuncDeclaration *self, Scope *sc)
```

Comment:
As far as i can [see][1] `objc_FuncDeclaration_semantic_validateSelector(..)` is a stub if `D_OBJC ` is enabled.

[1]: https://github.com/D-Programming-Language/dmd/pull/4321/files#diff-ea795c64edf6f31b4615d946696ac5f3R179